packages/env/src/ua-helper.ts

```diff
@@ -90,3 +90,25 @@ export function getUaHelper() {
 
   return uaHelper;
 }
+
+export function isElectron() {
```

Comment:
We already have `isDesktop` in `getEnvironment()`, or the `environment` variable, which right now has the same meaning to `isElectron`

Reply:
fixed